miniapps/tools/lor-transfer.cpp

```diff
@@ -95,13 +95,7 @@ int main(int argc, char *argv[])
    args.AddOption(&use_pointwise_transfer, "-t", "--use-pointwise-transfer",
                   "-no-t", "--dont-use-pointwise-transfer",
                   "Use pointwise transfer operators instead of L2 projection.");
-   args.Parse();
-   if (!args.Good())
-   {
-      args.PrintUsage(cout);
-      return 1;
-   }
-   args.PrintOptions(cout);
+   args.ParseCheck();
```

Comment:
Should double-check that the H1 transfer with this miniapp works with AMR meshes

Reply:
Just added in an AMR option, seems to be working OK with uniform refinement.  When only a subset of the elements are refined, the accuracy reduces, probably due to hanging nodes.  I assume this is expected behavior?  I'm not sure how MFEM handles hanging nodes on H1 meshes.

Reply:
I just meant to double check that `lor-transfer` gives correct results when running with e.g. `amr-quad.mesh` (there were issues with the restriction operators). It's fixed now. I reverted the AMR loop just to try to keep the miniapp simple.